src/Android/Accessibility/AccessibilityHelpers.cs

```diff
@@ -416,6 +420,24 @@ public static WindowManagerLayoutParams GetOverlayLayoutParams()
             return point;
         }
 
+        public static bool IsStatusBarExpanded(IEnumerable<AccessibilityWindowInfo> windows)
+        {
+            if(windows != null && windows.Count() > 0)
```

Comment:
Use .Any() instead of Count() > 0. It can be more performant depending on the underlying type of the IEnumerable.